token/services/identity/msp/idemix/lm.go

```diff
@@ -155,7 +156,11 @@ func (lm *LocalMembership) RegisterIdentity(id string, path string) error {
 	lm.resolversMutex.Lock()
 	defer lm.resolversMutex.Unlock()
 
-	if err := lm.storage.AddConfiguration(driver3.IdentityConfiguration{ID: id, URL: path}); err != nil {
+	if err := lm.storage.AddConfiguration(driver3.IdentityConfiguration{
+		ID:   id,
+		Type: "idemix",
```

Comment:
We could use an enum for the type since it can only be x509 or idemix. Same for the DB.

Reply:
will do